sdk/python/feast/constants.py

```diff
@@ -17,7 +17,8 @@
 # Maximum interval(secs) to wait between retries for retry function
 MAX_WAIT_INTERVAL: str = "60"
 
-AWS_LAMBDA_FEATURE_SERVER_IMAGE = "feastdev/feature-server:aws"
+# Dockerhub name for the AWS Lambda feature server docker image.
+AWS_LAMBDA_FEATURE_SERVER_IMAGE = "feastdev/feature-server:aws-v0.1"
```

Comment:
btw, where does this version come from? Just want to figure out how we will be bumping these versions. Don't want to randomize you, but normally these versions are assigned dynamically through CI. 

So basically when CI runs you push `feastdev/feature-server-aws:{GIT SHA}` and `feastdev/feature-server-aws:{SEMVER}` and if it's the latest stable image, then you also push `feastdev/feature-server-aws:latest`. You cant have `feastdev/feature-server:aws-latest`, there is no automatic selection of the latest image if the tag isn't exactly `latest`. 

Then above becomes
```suggestion
AWS_LAMBDA_FEATURE_SERVER_IMAGE = "feastdev/feature-server-aws"
```
which selects the latest stable image. Or you can pin it to the current version as well.

Reply:
Got it, synced with Achal about this.